### ISSUE TITLE
Brace being set on a new line for classes

### ISSUE THREAD

**Issue by ChloeDawn** [FabricMC/procyon#5]
Braces for classes are set on a new line rather than inline like other braces. This bug was experienced through Enigma. An example of what I am describing can be seen below.
```java
public class FooBar
{
	public void foobar() {
		if (Baz.condition()) {
			for (int i = 0; i < 10; ++i) {
			}
		}
	}
}
```

**Comment by Gegy** [comment]
I think this would be considered an Enigma issue -- the brace style is configurable through `JavaFormattingOptions`. This is the default but maybe Enigma should change the setting?

**Comment by ChloeDawn** [comment]
Oh, I assumed this was an issue at decompiler level, my bad. Can you transfer this to the Enigma repo?

**Comment by Gegy** [comment]
I don't believe I have the ability to do that, but could recreate the issue if needed?

**Comment by ChloeDawn** [comment]
Oh, I think only members/collaborators can do it, nvm. If you had permissions you would see this option in the right-hand sidebar:
![screenshot](https://i.imgur.com/yp4LFK4.png)

**Comment by Runemoro** [comment]
This was fixed a while ago in https://github.com/FabricMC/Enigma/pull/160.
